packages/modules/viewer-react/src/services/BaseInitializer.ts

```diff
@@ -290,5 +291,6 @@ export const getIModelAppOptions = (
         : viewerHome && `${viewerHome}/locales/{{lng}}/{{ns}}.json`,
     }),
     toolAdmin: options?.toolAdmin,
+    imodelClient: options?.imodelClient || new IModelHubClient(),
```

Comment:
```suggestion
    imodelClient: options?.imodelClient,
```

https://github.com/imodeljs/imodeljs/blob/master/core/frontend/src/FrontendHubAccess.ts#L39